Tests/js/views/services/assets/ez-navigationhubviewservice-tests.js

```diff
@@ -28,9 +29,35 @@ YUI.add('ez-navigationhubviewservice-tests', function (Y) {
                 }
             };
 
+            this.rootStruct = {'location': new Mock(), 'content': new Mock()};
+
+            Mock.expect(this.rootStruct.location, {
+                method: 'get',
+                args: [Mock.Value.Any]
```

Comment:
please avoid as much as possible `Mock.Value.Any`. AFAICS, in this test, there's will be only a call to `get('id')` on the locations and only a call to `get('mainLanguageCode')` on the contents so please configure the mocks for that.